src/libraries/System.Drawing.Common/tests/GdiplusTests.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing.Text;
+using Xunit;
+
+namespace System.Drawing.Tests
+{
+    public class GdiplusTests
+    {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX))]
```

Comment:
Or Windows?

Reply:
On Windows, System.Drawing.Common uses GDI+ as a backend instead of libgdiplus.

`GetLibgdiplusVersion` is a libgdiplus-only thing, and the versioning of libgdiplus is unrelated to the version numbers used by GDI+.

Long story short, this check won't work on Windows.

Reply:
I don't understand. You explicitly check for Windows and return true here:
https://github.com/qmfrederik/runtime/blob/1e29830cdf176f306289329bdcc5f6d75eb23253/src/libraries/Common/tests/System/Drawing/Helpers.cs#L33

Reply:
Thanks for pointing that out.

System.Drawing wraps around GDI+. GDI+ is the reference implementation, and libgdiplus the open-source re-implementation of GDI+ for Unix.

The GDI+ API as consumed by System.Drawing is very stable. libgdiplus on the other hand is still catching up with GDI+.

All unit tests for System.Drawing.Common should pass on GDI+. Some tests fail on libgdiplus, until somene fixes the underlying libgdiplus issue.

So what this method is `GetIsAtLeastLibgdiplus6` is really checking is:
- Are we on the GDI+ reference implementation (if so, tests should always pass)
- Are we on a recent enough version of libgdiplus (a test which was known to fail on libgdiplus, but for which a fix was issued in a recent version of libgdiplus).

So I guess the name is a bit off, and a better option would be `GetIsWindowsOrAtLeastLibgdiplus6` or `GetIsGdiPlusOrAtLeastLibgdiplus6`.

Let me know if you want me to rename the method, and which name you prefer :).

PS: Apologies for the long answer & if some of it was stating the obvious.

Reply:
Either proposed name is fine. Thanks.